karma.conf.js

```diff
@@ -0,0 +1,37 @@
+module.exports = function(config) {
+  config.set({
+
+    basePath: '.',
+
+    frameworks: ['browserify', 'jasmine'],
+
+    browserify: {
+      debug: true,
+      transform: [
+        ['babelify', {presets: ['es2015']}]
+      ]
+    },
+
+    files: [
+      'tests/*-spec.js'
```

Comment:
I like to have conf files in the `tests` folder so other node scripts (like various helpers and such) can be kept there too